packages/react/src/atomic/Button/CHANGELOG.md

```
@@ -2,6 +2,18 @@
 
 The Button component allows us to initiate actions, make state or page changes.
 
+## 2025-10-17 `2.10.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)